src/Smesh.jl

```diff
@@ -112,8 +232,112 @@ function voronoi_compute_neighbors(vertices, voronoi_vertices, voronoi_vertices_
                                                 n_vertices_voronoi::Cint,
                                                 n_elements_voronoi::Cint)::Cvoid
 
+    # Periodic neighbors
+    voronoi_compute_periodic_neighbors!(voronoi_neighbors, periodicity,
+                                        voronoi_vertices_coordinates, voronoi_vertices,
+                                        voronoi_vertices_interval)
+
     return voronoi_neighbors
 end
 
+"""
+    voronoi_compute_periodic_neighbors!(voronoi_neighbors, periodicity,
+                                        voronoi_vertices_coordinates, voronoi_vertices,
+                                        voronoi_vertices_interval)
+
+"""
+function voronoi_compute_periodic_neighbors!(voronoi_neighbors, periodicity,
+                                             voronoi_vertices_coordinates, voronoi_vertices,
+                                             voronoi_vertices_interval)
+    # Add neighboring elements if there are periodic boundaries
+    if !any(periodicity)
+        return nothing
+    end
+
+    standard_normal_vector_left = [[-1.0, 0.0], [0.0, -1.0]]
+    standard_normal_vector_right = [[1.0, 0.0], [0.0, 1.0]]
+    for dim in 1:2
+        if periodicity[dim]
+            # Initialize lists for boundary elements
+            boundary_elements_left = Int[]
+            boundary_faces_left = Int[]
+            boundary_elements_right = Int[]
+            boundary_faces_right = Int[]
+            for element in axes(voronoi_vertices_interval, 2)
+                face_index_start = voronoi_vertices_interval[1, element]
+                face_index_end = voronoi_vertices_interval[2, element]
+                for face_index in face_index_start:face_index_end
+                    if voronoi_neighbors[face_index] == 0 # Boundary face
+                        node1 = voronoi_vertices[face_index]
+                        if face_index < face_index_end
+                            node2 = voronoi_vertices[face_index + 1]
+                        else
+                            node2 = voronoi_vertices[face_index_start]
```

Comment:
This line is the only one that is not covered in the tests right now. The "last" face of each element does not seem to be a boundary face in all my exemplary meshes. So, there are different options:
- Ignore the missing line in the coverage
- Move lines 271–276 above if `if` query in line 270. (Unnecessary operations in simulation)
- Find a set of data points and therefore mesh where this line is used.
@simonechiocchetti @sloede What do you think about that?